packages/react-button/src/components/Button/useButtonClasses.ts

```diff
@@ -40,447 +40,450 @@ export const ButtonSizeVariants = {
       },
 
-      primary: {
-        background: 'var(--color-brand-background)',
-        borderColor: 'var(--color-brand-borderColor)',
-        contentColor: 'var(--color-brand-contentColor)',
-        iconColor: 'inherit',
-        forcedColorAdjust: 'none',
+      highContrast: {
+        background: 'WindowText',
+        borderColor: 'WindowText',
+        contentColor: 'Window',
+        focusColor: 'WindowText',
+        focusInnerColor: 'Window',
+        iconColor: 'Window',
 
         hovered: {
-          background: 'var(--color-brand-hovered-background)',
-          borderColor: 'var(--color-brand-hovered-borderColor)',
-          contentColor: 'var(--color-brand-hovered-contentColor)',
+          background: 'Highlight',
+          borderColor: 'Highlight',
+          contentColor: 'Window',
+          iconColor: 'Window',
         },
 
         pressed: {
-          background: 'var(--color-brand-pressed-background)',
-          borderColor: 'var(--color-brand-pressed-borderColor)',
-          contentColor: 'var(--color-brand-pressed-contentColor)',
-        },
-
-        highContrast: {
           background: 'WindowText',
           borderColor: 'WindowText',
           contentColor: 'Window',
-          focusColor: 'WindowText',
-          focusInnerColor: 'Window',
           iconColor: 'Window',
+        },
 
-          hovered: {
-            background: 'Highlight',
-            borderColor: 'Highlight',
-            contentColor: 'Window',
-            iconColor: 'Window',
-          },
+        disabled: {
+          background: 'Window',
+          borderColor: 'GrayText',
+          contentColor: 'GrayText',
+          iconColor: 'GrayText',
+        },
+      },
+    },
 
-          pressed: {
-            background: 'WindowText',
-            borderColor: 'WindowText',
-            contentColor: 'Window',
-            iconColor: 'Window',
-          },
+    // TODO: no references to palette.
+    ghost: {
+      background: 'transparent',
+      borderColor: 'transparent',
+      contentColor: palette?.neutralPrimary,
+      fontWeight: 'normal',
+      iconColor: palette?.themeDarkAlt,
+      forcedColorAdjust: 'none',
+
+      disabled: {
+        background: semanticColors?.disabledBackground,
+        contentColor: palette?.neutralTertiary,
+        iconColor: 'inherit',
+      },
 
-          disabled: {
-            background: 'Window',
-            borderColor: 'GrayText',
-            contentColor: 'GrayText',
-            iconColor: 'GrayText',
-          },
-        },
+      expanded: {
+        contentColor: palette?.themePrimary,
       },
 
-      // TODO: no references to palette.
-      ghost: {
-        background: 'transparent',
-        borderColor: 'transparent',
-        contentColor: palette?.neutralPrimary,
-        fontWeight: 'normal',
-        iconColor: palette?.themeDarkAlt,
-        forcedColorAdjust: 'none',
+      // focused: {
```

Comment:
I'm not sure how this is required, and also, it's not passing typings.

Reply:
Should be passing typings now if you merge with the latest changes from master.